iceoryx_examples/iceperf/iceoryx.cpp

```diff
@@ -21,8 +21,14 @@
 #include <thread>
 
 Iceoryx::Iceoryx(const iox::capro::IdString_t& publisherName, const iox::capro::IdString_t& subscriberName) noexcept
-    : m_publisher({"IcePerf", publisherName, "C++-API"}, iox::popo::PublisherOptions{1U})
-    , m_subscriber({"IcePerf", subscriberName, "C++-API"}, iox::popo::SubscriberOptions{1U, 1U})
+    : Iceoryx(publisherName, subscriberName, "C++-API")
+{
+}
+Iceoryx::Iceoryx(const iox::capro::IdString_t& publisherName,
+                 const iox::capro::IdString_t& subscriberName,
+                 const iox::capro::IdString_t& eventName) noexcept
```

Comment:
I had to think a bit what `eventName` means. I guess this comes from using `eventId` when we talk about this part of the `ServiceDescription`

Reply:
/// @brief construction of the capro service description using fixed strings to create an event service description
    ServiceDescription(const IdString_t& service,
                       const IdString_t& instance,
                       **const IdString_t& event**,
                       ClassHash m_classHash = {0U, 0U, 0U, 0U},
                       Interfaces interfaceSource = Interfaces::INTERNAL) noexcept;